ws/templates/help/leaders/ws_gear.html

```diff
@@ -49,87 +49,94 @@ <h4>Special Gear</h4>
   </ul>
 </div>
 
-<h2 id="group">Group Gear</h2>
+<h2 id="group">Group Safety Gear</h2>
 <p>
   Trip leaders must ensure that collectively their group carries the following gear according to trip type.
 </p>
 
 <div class="well" id="a_trip">
   <h4>A Trip</h4>
   <ul>
-    <li>MITOC first-aid kit</li>
-    <li>Extra clothing</li>
-    <li>Extra food</li>
-    <li>Duct tape</li>
-    <li>Parachute cord or webbing</li>
-    <li>Foam pad (optional)</li>
+    <li>One MITOC first aid kit,</li>
+    <li>Extra layers (clothing),</li>
+    <li>Extra food and water,</li>
+    <li>Two foam pads,</li>
+    <li>Duct tape or climbing tape, and</li>
+    <li>Parachute cord or webbing.</li>
   </ul>
 </div>
 
 <div class="well" id="b_trip">
   <h4>B Trip</h4>
   <p>
-    Everything for an A Trip plus ...
+    
   </p>
   <ul>
-    <li>2 winter sleeping bags rated for the forecasted low temp of the night at the elevations you expect to travel to (check on Mountain Forecast)</li>
-    <li>1 foam pad for every 2 people</li>
-    <li>Winter-rated stove with pot, lid, fuel (min 22oz, 33oz recommended)</li>
-    <li>2 types of fire starters (flint + steel, waterproof matches, lighter)</li>
-    <li>2 Bivvies</li>
-    <li>
-      2 Outdoor Research Alpine bivvies
-      <ul>
-        <li>Or one MITOC pack of 3 large yellow plastic bags</li>
-        <li>Or one extra shelter</li>
-      </ul>
-    </li>
-
-    <li>
-      Shelter/Tent
-      <ul>
-        <li>MITOC beta-mid (2 person pyramid tarp)</li>
-        <li>Or MITOC beta-light (2 person pyramid tarp)</li>
-        <li>Or MITOC mega-light (4 person pyramid tarp)</li>
-        <li>Or MITOC bothy bag (Terra Nova Bothy 8)</li>
-        <li>Or your own 3-season tent</li>
-      </ul>
-    </li>
+    <li>All the group gear required for an A Trip plus ...</li>
+    <li>Two sleeping bags (one winter (15F) rated and one three-season (35F) rated minimum),</li>
+    <li>One bivy or bothy bag - A large MITOC tarp, an actual bothy bag, or a bivy bag such as the OR Alpine Bivy),</li>
+    <li>One shelter - A three-season or four-season tent and if weather conditions permit a bothy or tarp, and</li>
+    <li>A winter-rated stove with pot, lid, fuel, and fire starters (optional).</li>
   </ul>
 </div>
 
 <div class="well" id="c_trip">
   <h4>C Trips</h4>
-  <p>
-  Everything for a B Trip plus...
-  </p>
   <ul>
-    <li>Mountaineering Shelter/Tent: BD First Light, MHW Direkt 2, Trango, or other 4-season mountaineering tent (instead of B Trip shelter)</li>
-    <li>Shovel (optional)</li>
-    <li>Wands (optional)</li>
+    <li>All the group gear required for a B Trip with the following modifications ...</li> 
```

Comment:
This looks like a heading, not a list item (the rest of the following content should be `li`). Put this into the since-deleted `<p>` element above, and keep the rest of the list as-is?